src/backend/dx12/src/command.rs

```diff
@@ -166,15 +166,18 @@ impl PipelineCache {
         }
     }
 
-    fn bind_descriptor_sets<'a, T>(
+    fn bind_descriptor_sets<'a, I, J>(
         &mut self,
         layout: &n::PipelineLayout,
         first_set: usize,
-        sets: T,
+        sets: I,
+        offsets: J,
     ) -> [*mut d3d12::ID3D12DescriptorHeap; 2]
     where
-        T: IntoIterator,
-        T::Item: Borrow<n::DescriptorSet>,
+        I: IntoIterator,
+        I::Item: Borrow<n::DescriptorSet>,
+        J: IntoIterator,
+        J::Item: Borrow<com::DescriptorSetOffset>,
```

Comment:
Actually, I don't think there is much value in going with `Borrow<>` here, since the offsets are easily copyable. Mind changing it to just `IntoIterator<Item = com::DescriptorSetOffset>`?

Reply:
sure, beside that I'm done with the PR from my side as far as I can see.

Reply:
great! i'm also ready with Metal side, just blocked on that `IntoIterator` change

Reply:
hold up, actually not that much a fan of the `intoIterator` change, makes using slices difficult/impossible as far as I can see.

Reply:
aha, good point